core/services/service-management/services/ServiceImagesService.js

```diff
@@ -0,0 +1,91 @@
+const { Readable } = require("stream");
+const { GridFSBucket } = require("mongodb");
+const generateUuid = require("@diva/common/generateUuid");
+const { servicesMongoDbConnector } = require("../utils/mongoDbConnectors");
+const {
+  imageNotFoundError,
+  wrongImageFormatError,
+} = require("../utils/errors");
```

Comment:
@anatoliynovo 
cant't find this errors in [utils/errors](https://github.com/FraunhoferISST/diva/blob/b3d3af75b35bd9aed3237c17b77e99cff169a574/core/services/service-management/utils/errors.js), forgotten?